hieradata/cluster/pukem.yaml

```diff
@@ -0,0 +1,6 @@
+---
+clustershell::groupmembers:
+  pukem: {group: "pukem", member: "pukem[01-04]"}
+profile::core::ospl::enable_rundir: true
+profile::core::k8snode::enable_dhcp: true
```

Comment:
this is just in case the cluster uses Multus or any CNI plugin (macvlan, etc.)

Reply:
correct